ui/admin/app/components/form/worker/create-worker-led/index.hbs

```diff
@@ -12,43 +12,61 @@
   <h3>{{t 'resources.worker.form.steps.1.title'}}</h3>
   <div class='worker-create-section'>
     {{#if (feature-flag 'byow-pki-hcp-cluster-id')}}
-      <form.input
-        @name='cluster_id'
+      <Hds::Form::TextInput::Field
+        name='cluster_id'
         @value={{this.clusterID}}
-        @label={{t 'resources.worker.form.cluster_id.label'}}
-        @helperText={{t 'resources.worker.form.cluster_id.help'}}
-        @linkText={{t 'actions.learn-more'}}
-        @link={{doc-url 'worker.manage-workers'}}
         placeholder='69b6ddb3-ffec-42ab-a994-f43a6519470a'
-      />
+        as |F|
+      >
+        <F.Label>{{t 'resources.worker.form.cluster_id.label'}}</F.Label>
+        <F.HelperText>
+          {{t 'resources.worker.form.cluster_id.help'}}
+          <Hds::Link::Inline @href={{doc-url 'worker.manage-workers'}}>
+            {{t 'actions.learn-more'}}
+          </Hds::Link::Inline>
+        </F.HelperText>
+      </Hds::Form::TextInput::Field>
     {{/if}}
 
-    <form.input
-      @name='ip_address'
+    <Hds::Form::TextInput::Field
+      name='ip_address'
       @value={{this.ipAddress}}
-      @label={{t 'resources.worker.form.ip_address.label'}}
-      @helperText={{t 'resources.worker.form.ip_address.help'}}
-      @linkText={{t 'actions.learn-more'}}
-      @link={{doc-url 'worker'}}
       placeholder='worker1.example.com'
-    />
-
-    <form.input
-      @name='config_file_path'
+      as |F|
+    >
+      <F.Label>{{t 'resources.worker.form.ip_address.label'}}</F.Label>
+      <F.HelperText>
+        {{t 'resources.worker.form.ip_address.help'}}
+        <Hds::Link::Inline @href={{doc-url 'worker'}}>
+          {{t 'actions.learn-more'}}
+        </Hds::Link::Inline>
+      </F.HelperText>
+    </Hds::Form::TextInput::Field>
+
+    <Hds::Form::TextInput::Field
+      name='config_file_path'
       @value={{this.configFilePath}}
-      @label={{t 'resources.worker.form.config_file_path.label'}}
-      @helperText={{t 'resources.worker.form.config_file_path.help'}}
       placeholder='/home/ubuntu/boundary'
-    />
+      as |F|
+    >
+      <F.Label>{{t 'resources.worker.form.config_file_path.label'}}</F.Label>
+      <F.HelperText>
+        {{t 'resources.worker.form.config_file_path.help'}}
+      </F.HelperText>
+    </Hds::Form::TextInput::Field>
 
     {{#if (not (feature-flag 'byow-pki-hcp-cluster-id'))}}
-      <form.input
-        @name='initial_upstreams'
+      <Hds::Form::TextInput::Field
```

Comment:
Same question: https://github.com/hashicorp/boundary-ui/pull/2151/files#r1503424846

Reply:
@calcaide https://github.com/hashicorp/boundary-ui/pull/2151#discussion_r1504472569

Reply:
I see you add `@isOptional`